src/Zicht/Bundle/AdminBundle/Doctrine/CustomSortProxyQuery.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @copyright Zicht Online <https://zicht.nl>
+ */
+
+namespace Zicht\Bundle\AdminBundle\Doctrine;
```

Comment:
Ik zou deze wellicht in de namespace `Sonata` verwachten :thinking:

Reply:
Done :+1: